sim/common/sod/item_effects/phase_8.go

```diff
@@ -1039,6 +1040,31 @@ func init() {
 		character.ItemSwap.RegisterActive(SoporificBlade)
 	})
 
+	// https://www.wowhead.com/classic-ptr/item=240843/starters-pistol
+	// Equip: Firing a regular ranged attack at a target prepares you for battle, increasing your Defense by 20 and melee attack speed by 10% for 15 sec.
+	core.NewItemEffect(StartersPistol, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		buffAura := character.RegisterAura(core.Aura{
+			ActionID: core.ActionID{SpellID: 1231266},
+			Label:    "En Garde!",
+			Duration: time.Second * 15,
+		}).AttachStatBuff(stats.Defense, 20).AttachMultiplyMeleeSpeed(&character.Unit, 1.10)
+
+		triggerAura := core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
+			ActionID:   core.ActionID{SpellID: 1231267},
```

Comment:
🔨 we don't really need to add action IDs to most triggers like this. It just ends up adding bloat in the UI